symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/impl/DatafeedServiceV1.java

```diff
@@ -83,7 +84,7 @@ public void start() throws AuthUnauthorizedException, ApiException {
       do {
         readDatafeed();
       } while (this.started.get());
-    } catch (AuthUnauthorizedException | ApiException | NestedRetryException exception) {
+    } catch (AuthUnauthorizedException | ApiException | NestedRetryException | MessageValidationException exception) {
```

Comment:
should we make it a subclass of ApiException so that we don't to change the code everywhere each time we add a new type?